src/features/HelpAndKnowledge/Articles/UsingMilestoneLibrary/index.tsx

```diff
@@ -0,0 +1,201 @@
+                {t('description')}
+              </p>
+
+              {/* Suggested milestones */}
+              <div>
+                <h2 className="margin-top-0 margin-bottom-2">
+                  {t('suggestedMilestones.title')}
+                </h2>
+                <p>{t('suggestedMilestones.description')}</p>
+                <div className="border-1px radius-md border-gray-10 shadow-3">
+                  <img
+                    alt={t('suggestedMilestones.altText')}
+                    src={suggestedMilestoneLibraryImg}
+                  />
+                </div>
+                <p>{t('suggestedMilestones.caption')}</p>
+              </div>
+
+              {/* Browsing all available milestones */}
+              <div>
+                <h2 className="margin-top-5 margin-bottom-2">
+                  {t('browsingAllAvailableMilestones.title')}
+                </h2>
+                <p>{t('browsingAllAvailableMilestones.description')}</p>
+
+                <div className="border-1px radius-md border-gray-10 shadow-3">
+                  <img
+                    src={allMilestoneLibraryImg}
+                    alt={t('browsingAllAvailableMilestones.altText')}
+                  />
+                </div>
+                <p>{t('browsingAllAvailableMilestones.caption')}</p>
+                {browsingAllAvailableMilestonesList.map(item => (
+                  <div key={item.point}>
+                    <h3 className="margin-bottom-1">{t(item.point)}</h3>
+                    <p className="margin-y-0">{t(item.text)}</p>
+                  </div>
+                ))}
+              </div>
+
+              {/* Adding a milestone to your MTO */}
+              <div>
+                <h2 className="margin-top-5 margin-bottom-2">
+                  {t('addingMilestoneToMto.title')}
+                </h2>
+                <ProcessList>
+                  {/* step 1 */}
+                  <ProcessListItem className="maxw-none">
+                    <ProcessListHeading type="h3">
+                      {t('addingMilestoneToMto.step1.title')}
+                    </ProcessListHeading>
+                    <p>{t('addingMilestoneToMto.step1.description')}</p>
+
+                    <UswdsReactLink
+                      aria-label="Redirect to using table actions article"
+                      to="using-table-actions"
+                      className="display-block margin-y-05"
+                    >
+                      {t('addingMilestoneToMto.step1.link1')}{' '}
+                      <Icon.ArrowForward className="margin-left-1 top-05 text-primary" />
+                    </UswdsReactLink>
+                    <UswdsReactLink
+                      aria-label="Redirect to starting mto for a new model article"
+                      to="using-table-actions"
+                      className="display-block margin-y-05"
+                    >
+                      {t('addingMilestoneToMto.step1.link2')}{' '}
```

Comment:
I think we both made the same mistake haha.  This should be `starting-mto-matix`